net/netlink/netlink_sockif.c

```diff
@@ -595,7 +593,7 @@ static int netlink_poll(FAR struct socket *psock, FAR struct pollfd *fds,
       if (revents != 0)
         {
           fds->revents = revents;
-          nxsem_post(fds->sem);
+          poll_notify_nocheck(&fds, 1, 0);
```

Comment:
Should it be
```suggestion
          poll_notify_nocheck(&fds, 1, revents);
```
?

Reply:
What about this place. It seems to be different from other similar places

Reply:
Yes, and i will delete the api poll_notify_nocheck() (only left poll_notify()), because we must check fds->events before set the fds->revents, otherwise the app may poll success but the fds->revent is not the app wants.

Reply:
Done